pkg/broker/handler/fanout.go

```diff
@@ -176,7 +176,6 @@ func (p *FanoutPool) SyncOnce(ctx context.Context) error {
 
 		// Start the handler with broker key in context.
 		hc.Start(handlerctx.WithBrokerKey(ctx, b.Key()), func(err error) {
-			// We will anyway get an error because of https://github.com/cloudevents/sdk-go/issues/470
```

Comment:
Is this change unrelated?

Reply:
unrelated. Was looking at code and saw that TODO but that bug was fixed so I removed the comment. Let me know if you want it back

Reply:
feel free to unresolved if you want this comment back in